config/cp4waiops/install-aimgr/values.yaml

```diff
@@ -43,7 +43,7 @@ spec:
     ## A channel defines a stream of updates for an Operator and is used to roll out updates for subscribers.
     ## For example, if you want to install AI Manager 3.2, the channel should be v3.2
     ##
-    channel: v3.4
+    channel: v3.5
```

Comment:
The event manager channel may also need to be updated to reflect the latest version used in CP4WAIOps v3.5.